src/components/VaccinationBlock/LinkButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Button, ChevronRightIcon } from './VaccinationBlock.style';
+
+type ButtonProps = React.ComponentProps<typeof Button>;
+
+const LinkButton: React.FC<ButtonProps> = ({
```

Comment:
This is kind of new, I didn't know that you can grab the type of the component that you are importing and then use it. This is handy because then the `LinkButton` can receive the same props than `Button` (which in turn, can receive the same props than `MuiButton`)

Reply:
ooo interesting